db/schema.rb

```diff
@@ -91,6 +91,17 @@
     t.index ["user_id"], name: "index_journeys_on_user_id", using: :btree
   end
 
+  create_table "logs", force: :cascade do |t|
+    t.integer  "minutes"
```

Comment:
Agreed. I will change the table name by `class_logs`.  

Then I will move the `minutes_paid` and `hours_paid` into a similar table that the one you created of `space_orders` in Kuina (included the stripe stuff). The issue is that in this business there are not orders as such, as we hope that that parents will pay 'as they go' (depending on the classes that a teacher will do in a given week). I was thinking to call that table: `invoices`. What do you think?